ballsdex/packages/balls/cog.py

```diff
@@ -206,11 +206,11 @@ async def list(
             ball_txt = countryball.country if countryball else ""
             if user_obj == interaction.user:
                 await interaction.followup.send(
-                    f"You don't have any {ball_txt} {settings.collectible_name} yet."
+                    f"You don't have any {ball_txt} {settings.collectible_name}s yet."
                 )
             else:
                 await interaction.followup.send(
-                    f"{user_obj.name} doesn't have any {ball_txt} {settings.collectible_name} yet."
+                    f"{user_obj.name} doesn't have any {ball_txt} {settings.collectible_name}s yet."
```

Comment:
This line became too long for the code style
```suggestion
                    f"{user_obj.name} doesn't have any "
                    f"{ball_txt} {settings.collectible_name}s yet."
```